backend/src/osm_tests/README.md

```diff
@@ -0,0 +1,20 @@
+# OSM unit tests
+
+Importing existing modal filters and turn restrictions from OSM can be very complicated. This directory contains minimal synthetic OSM inputs to test different situations.
+
+## Creating a unit test
+
+1.  Open the iD editor and zoom in on an empty patch of ocean, [like here](https://www.openstreetmap.org/edit#map=17/55.704282/-0.110917)
```

Comment:
Thanks for writing this up! I was not clear on how we'd actually use this in practice. For some reason, it didn't occur to me to use an empty ocean.